### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Manish-Syal123**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resource(s)
### Description
Ansible Cheat Sheet
### Why is this valuable (or not)?
Free
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Manish-Syal123** [comment]
Sir I have One doubt that is  this repo is not participating in Hacktoberfest , because I merged 2 PR from this repo but those are not counted in my Hacktoberfest profile .
They are saying that : 
EXCLUDED?
YOUR PR/MR WAS SUBMITTED TO A REPOSITORY THAT HAS BEEN EXCLUDED FROM HACKTOBERFEST AS IT DOES NOT FOLLOW THE VALUES OF THE EVENT.

abc
Screenshot 2022-10-20 232542

**Comment by eshellman** [comment]
Thank you for your report about the possible octoberfest exclusion of free-programming-books. I have emailed to the organizers about this. I suggest you move your report to issues where others will see it. you might also report to hacktoberfest@digitalocean.com I suspect that we were reported as retaliation for reporting spam.

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -50,6 +51,15 @@
 * [Webpack](#webpack)
 
 
+### Ansible
+
+* [Ansible Basic Cheat Sheet](https://intellipaat.com/blog/tutorial/devops-tutorial/ansible-basic-cheat-sheet) - Intellipaat (HTML)
+* [Ansible Cheat Sheet](https://sites.google.com/site/mrxpalmeiras/ansible/ansible-cheat-sheet) - Mrxpalmeiras (HTML)
+* [Ansible Cheat Sheet — A DevOps Quick Start Guide](https://medium.com/edureka/ansible-cheat-sheet-guide-5fe615ad65c0) - Medium (HTML, PDF)
```
[the remark this answers is not in the document]
Comment (Manish-Syal123) [reply more/free-programming-cheatsheets.md]: resolved

[the remark this answers is not in the document]
Comment (Manish-Syal123) [reply more/free-programming-cheatsheets.md]: Done 👍

**Comment by eshellman** [comment]
issue #8671

**Pull request merged**